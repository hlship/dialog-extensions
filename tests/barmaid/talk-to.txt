%% Exercise implicit greeting and expressing ignorance (the immortality topic is
%% linked to a later quip.)
talk to bar maid about immortality
