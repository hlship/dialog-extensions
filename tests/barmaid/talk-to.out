

It is a long and riotous evening, full of unlikely stories and tall tales. But
now, most of the patrons have gone away to their rooms to sleep, or have passed
out before the fire. Even the two black bitch pups are curled on the hearth-
stone, snuffling through tiny wet noses, and pawing the air in sleep. Now is the
time to find out whether the rumors that brought you to this neighborhood are
true.

The Inn
The light is low and red. Above the fireplace hangs a map of this part of the
Empire, and on the adjacent wall is the crimson and silver banner of Richard the
Rose. The barmaid works her way from table to table, clearing away dishes and
mugs.

> talk to bar maid about immortality
You say hello to the Bar Maid.

"I don't know anything about that," says the Bar Maid.

You could ask whether the rumors tell truly or ask where she comes from.

> 
